### DISCUSSION TITLE
Certificate Attribute not Passed through to Radius server

### DISCUSSION THREAD

**Discussion by blunsfordIT** [radsecproxy/radsecproxy#110] [General]
We setup 802.1x to connect to radsecproxy and radsecproxy connects to clearpass. When radsec connects directly to clearpass it sends the certificate attributes of the users machine.
When routing through the radsecproxy the certificate issuer, expiration attributes don't appear on the clearpass server as inputs. It is like they are stripped and only certain attributes are sent (computername, client-mac-vendor) . 

Is there a way to allow all or certain important attributes from the machine to pass through the radsecproxy to the clearpass server?

**Comment by fmauchle** [comment]
Generally, radsecproxy should not touch the radius attributes (other than message-authenticator) unless you explicitly instruct it to do so with a `rewrite {}` config (to add, modify or drop radius attributes). Could you provide an example which radius attribute(s) is missing? (radius attribute ID and example content)
